packages/explicit-constructed-response/src/main.js

```diff
@@ -53,29 +56,44 @@ export class Main extends React.Component {
 
   render() {
     const { showCorrectAnswer, value } = this.state;
-    const { classes, mode, prompt, rationale, teacherInstructions, note, showNote, env } = this.props;
+    const {
+      classes,
+      mode,
+      prompt,
+      rationale,
+      teacherInstructions,
+      note,
+      showNote,
+      env,
+      animationsDisabled,
+      alwaysShowCorrect
+    } = this.props;
     const { role } = env || {};
     const displayNote = (showCorrectAnswer || mode === 'view' && role === 'instructor') && showNote && note;
+    const mainClasses = alwaysShowCorrect ? classNames(classes.mainContainer, classes.noBorderColor)
+      : classes.mainContainer;
 
     return (
-      <div className={classes.mainContainer}>
+      <div className={mainClasses}>
         {
           teacherInstructions && hasText(teacherInstructions) && (
             <div className={classes.collapsible}>
-              <Collapsible
-                labels={{ hidden: 'Show Teacher Instructions', visible: 'Hide Teacher Instructions' }}
-              >
-                <div dangerouslySetInnerHTML={{ __html: teacherInstructions }}/>
-              </Collapsible>
+              {!animationsDisabled ? <Collapsible
+                  labels={{ hidden: 'Show Teacher Instructions', visible: 'Hide Teacher Instructions' }}
+                >
+                  <div dangerouslySetInnerHTML={{ __html: teacherInstructions }}/>
```

Comment:
Let's move this outside (the div with html) so if anything has to change, we can change in a single place.